src/flow-manager.c

```diff
@@ -378,8 +378,7 @@ static void FlowManagerHashRowTimeout(FlowManagerTimeoutThread *td, Flow *f, SCT
  * \param td FM timeout thread instance
  * \param f head of the evicted list
  */
-static void FlowManagerHashRowClearEvictedList(
-        FlowManagerTimeoutThread *td, Flow *f, SCTime_t ts, FlowTimeoutCounters *counters)
+static void FlowManagerHashRowClearEvictedList(FlowManagerTimeoutThread *td, Flow *f)
```

Comment:
Looks good, but I see other with `-Wunused-parameter`
- BypassedFlowUpdate
- FlowUpdateTtlTS
- FlowBypassedTimeout

Reply:
Thanks. Will use this while compiling.
Q: Why don't we have this in CI?